src/arrayMethodSort.js

```diff
@@ -4,8 +4,29 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    const defaultFunc = (a, b) => {
+      const strA = String(a);
+      const strB = String(b);
+
+      return strA < strB ? -1 : strA > strB ? 1 : 0;
+    };
+
+    const comparator =
+      typeof compareFunction === 'function' ? compareFunction : defaultFunc;
```

Comment:
The line break between `comparator =` and the ternary operation is unnecessary and reduces readability. Consider placing the entire assignment on a single line for clarity.